src/lib/dnssd/Advertiser_ImplMinimalMdns.cpp

```diff
@@ -106,12 +155,8 @@ class AdvertiserMinMdns : public ServiceAdvertiser,
     AdvertiserMinMdns() : mResponseSender(&GlobalMinimalMdnsServer::Server())
     {
         GlobalMinimalMdnsServer::Instance().SetQueryDelegate(this);
-        for (auto & allocator : mQueryResponderAllocatorOperational)
-        {
-            mResponseSender.AddQueryResponder(allocator.GetQueryResponder());
-        }
-        mResponseSender.AddQueryResponder(mQueryResponderAllocatorCommissionable.GetQueryResponder());
-        mResponseSender.AddQueryResponder(mQueryResponderAllocatorCommissioner.GetQueryResponder());
+        LogErrorOnFailure(mResponseSender.AddQueryResponder(mQueryResponderAllocatorCommissionable.GetQueryResponder()));
```

Comment:
We shouldn't be using LogErrorOnFailure in device code, since it uses `__FILE__`.  This applies in various places in this code.